conf/uvek.conf.part

```diff
@@ -1131,6 +1131,24 @@ source src_ch_bfe_sachplan-geologie-tiefenlager_vernehmlassung_3 : def_searchabl
         FROM bfe.geologische_tief_vernehm_pm_pt
 }
 
+source src_ch_bav_anlagen-schienengueterverkehr : def_searchable_features
+ {
+     sql_db = uvek_dev
+     sql_query = \
+         SELECT bgdi_id as id \
+             , name as label \
+             , 'feature' as origin \
+             , remove_accents(coalesce(dst_nr::text, ' ')||' '||coalesce(name, ' '))||' '||coalesce(dst_abk, ' ') as  detail \
```

Comment:
please use concat_ws, this will ensure we don't have trailing spaces in the indices